docs/topics/testing/advanced.txt

```diff
@@ -580,10 +580,15 @@ and tear down the test suite.
     If ``shuffle`` is an integer, test cases will be shuffled in a random order
     prior to execution, using the integer as a random seed. If ``shuffle`` is
     ``None``, the seed will be generated randomly. In both cases, the seed will
-    be logged to the console and set to ``self.shuffle_seed`` prior to running
-    tests. This option can be used to help detect tests that aren't properly
-    isolated. :ref:`Grouping by test class <order-of-tests>` is preserved when
-    using this option.
+    be logged and set to ``self.shuffle_seed`` prior to running tests. This
+    option can be used to help detect tests that aren't properly isolated.
+    :ref:`Grouping by test class <order-of-tests>` is preserved when using this
+    option.
+
+    ``logger`` can be used to pass a Python `Logger object`_. If provided, the
+    Logger will be used to log messages instead of printing to the console. The
+    Logger object will respect its logging level rather than the configured
+    ``verbosity``.
```

Comment:
Missing a `..versionadded`?

I also wonder if a link to the refreshed logging docs could be helpful?

https://github.com/django/django/blob/main/docs/howto/logging.txt

Reply:
Thanks for your comments, @smithdc1. I incorporated all of them except for linking to the how-to docs. I'm not sure it's needed for this niche argument, and I didn't see a way to add a link without adding verbiage like, "For information on configuring and using logging, see ...," which seemed like clutter here. If you have something in mind that's unobtrusive, feel free to suggest.

Reply:
That's fine, it was just a thought. 👍